package.json

```diff
@@ -13,42 +14,86 @@
     "eslint-plugin-react-hooks": "^2.1.2",
     "mocha": "^5.2.0",
     "node-fetch": "^2.6.0",
+    "nodemon": "^1.19.4",
+    "path": "^0.12.7",
     "rpc-websockets": "4.0.0",
     "tempfile": "^3.0.0",
-    "typescript": "^3.6.4"
+    "typescript": "^3.6.4",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
   },
   "dependencies": {
     "@apollo/react-hooks": "^3.1.2",
+    "@apollo/react-ssr": "^3.1.3",
+    "@babel/core": "^7.6.4",
```

Comment:
Should these go into devDependencies instead?

Reply:
This was all just local testing, so they won't exist at all.